src/tests/cli_tests.py

```diff
@@ -0,0 +1,454 @@
+#!/usr/bin/python
```

Comment:
On some newer OSes python == python3, so I usually go with `#!/usr/bin/env python2` (or even `#!/usr/bin/python2`).
Is this OK or does it cause any issues on macOS?